### DISCUSSION TITLE
2.3.110 Hotfix Failing on Airgap Deployment

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#7729] [Unsupported Versions]
Hello all,

I have an distributed airgap deployment of SO at the 2.3.110 version. When I attempt to apply the hotfix I receive the following error in the soup.log:

**Package salt-master-3004-1.el7.noarch already installed and latest version
Package salt-minion-3004-1.el7.noarch already installed and latest version
Nothing to do
[1;32m *  INFO[0m: Running install_centos_stable_post()
[1;32m *  INFO[0m: Running install_centos_check_services()
[1;32m *  INFO[0m: Salt installed! (0)
Applying yum versionlock for Salt.

Loaded plugins: versionlock
Adding versionlock on: 0:salt-master-3004-1.el7
Adding versionlock on: 0:salt-minion-3004-1.el7
Adding versionlock on: 0:salt-3004-1.el7
versionlock added: 3
Checking if Salt was upgraded.

Salt upgrade failed. Check of indicators of failure in /root/soup.log.
Once the issue is resolved, run soup again.
Exiting.**

When I run salt --version & salt-master --version it shows 3004, not 3004.1.

Here is the output of salt-master --versions-history:

![salt](https://user-images.githubusercontent.com/84025129/162002864-f1a6800a-1f69-4e16-bf32-f41339e25d7b.JPG)

The 2.3.110 install is clean, no additional packages have been updated or added since the installation. I'm sure it's a simple fix, but could someone point me in the right direction?

TYIA!

**Comment by TOoSmOotH** [comment]
Ok you will need to do the following:

Mount the hotfix disk:

mkdir /tmp/update
sudo mount /dev/cdrom /tmp/update

Copy the updated packages:

sudo rsync -shav /tmp/update/Packages/* /nsm/repo/
sudo createrepo /nsm/repo

The just rerun soup.

**Reply by scarl3t0** [reply]
I tried that fix and got the same result as before. Salt failed to upgrade, soup.log looks the same.

**Comment by TOoSmOotH** [comment]
@scarl3t0 Here is the latest hotfix ISO. Please make sure your salt-master service is running before souping.

https://download.securityonion.net/file/securityonion/securityonion-2.3.110-20220407.iso

Let me know how this goes
